assets/blocks/course-outline/module-block/index.js

```diff
@@ -29,16 +28,57 @@ registerBlockType( 'sensei-lms/course-outline-module', {
 			type: 'string',
 			default: '',
 		},
+		mainColor: {
+			type: 'string',
+		},
+		customMainColor: {
+			type: 'string',
+		},
+		textColor: {
+			type: 'string',
+		},
+		customTextColor: {
+			type: 'string',
+		},
+		className: {
+			type: 'string',
+		},
+		customClassName: {
+			type: 'string',
+		},
 	},
-	edit( props ) {
-		return <EditModuleBlock { ...props } />;
+	example: {
+		attributes: {
+			title: 'Module',
+			description: 'About Module',
+			innerBlocks: [
```

Comment:
I think it should be a sibling of the `attributes`, right? It didn't load the lesson in the preview, but in the Outline it worked.

Reply:
Ah, yes, thanks! And this also solves the issue with the innerblocks template! Fixing in 901943c